src/rules/orderedImportsRule.ts

```diff
@@ -109,6 +109,19 @@ function findUnsortedPair(xs: ts.Node[], transform: (x: string) => string): [ts.
     return null;
 }
 
+function sortByKey<T>(xs: T[], getSortKeyCallback: (x: T) => string): T[] {
```

Comment:
i'd recommend just calling this `getSortKey`, since it returns a string and not a function